mypy.ini

```diff
@@ -15,6 +15,9 @@ ignore_missing_imports = True
 [mypy-juriscraper.*]
 ignore_missing_imports = True
 
+[mypy-localflavor.*]
```

Comment:
Im just going to assume you know what you are doing here.

Reply:
This is what you have to do for each library that lacks type hints to tell mypy to ignore it.